collectors/ebpf.plugin/ebpf_socket.h

```diff
@@ -41,7 +41,7 @@ typedef enum ebpf_socket_idx {
     NETDATA_SOCKET_COUNTER
 } ebpf_socket_index_t;
 
-#define NETDATA_SOCKET_GROUP "Socket"
+#define NETDATA_SOCKET_GROUP "Network Viewer"
```

Comment:
I'd say that "Network Viewer" is not an appropriate name for this section. "Socket" or "Network" is more suitable.